packages/server/src/modules/user/resolvers.js

```diff
@@ -28,7 +29,7 @@ export default pubsub => ({
         }
 
         const e = new FieldError();
-        e.setError('user', translate(universalCookies.get('lang'), 'user', 'accessDenied'));
+        e.setError('user', i18n.t('user:accessDenied'));
```

Comment:
This is wrong - you are still using global object, you need to use the object provided to you by i18next-express-middleware in each request